ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/map.view.js

```diff
@@ -287,18 +287,23 @@ module.exports = Marionette.LayoutView.extend({
     this.toolbarPanZoom.show(new PanZoomView())
   },
   addHome: function() {
-    // TODO combine home and save buttons into a "split button dropdown" once this is refactored to React: DDF-4327
+    const self = this
```

Comment:
✏️ 	If you use an arrow function in `template()` you can use `this` directly and remove this line